docs/adrs/005-release-strategy.md

```diff
@@ -0,0 +1,102 @@
+- E2E tests may use canary or production landscapes.
+  In case of multiple landscapes:
+   - GitHub matrix builds can be used to easily testing against multiple landscapes
+   - For any differences between landscapes, test toggles need to be considered (e.g. `@EnabledIfSystemProperty`)
+   - Such toggles come with a bit of additional maintenance cost, as they need to be removed once the feature is released to all landscapes
+
+## Release
+
+For releasing new AI SDK versions, several points during the AI Core release process can be considered:
+
+- Release to any canary landscape
+- Release to all canary landscapes
+- Release to any production landscape
+- Release to all production landscapes
+
+Furthermore, the following boundary conditions need to be considered:
+
+- Releases to canary are usually deployed to production after 1-2 weeks
+- Releases to canary are not guaranteed to reach production
+- Different landscapes (both canary and prod) may host different versions of AI Core or different feature-toggle configurations
+- Historically, production EU10 is the most feature-complete landscape
+
+Finally, please note that in case of daily delivery the term _release_ still holds, typically referring to the enablement of a feature toggle rather than a physical deployment.
+However, here there would be more freedom in when a feature toggle is enabled for which landscape, and the timelines can be much shorter than weeks.
+
+## Decision
+
+We decide as follows:
+
+> 1. New AI SDK versions are released roughly every 2 weeks, shortly after new AI Core versions have been released to _production EU10_.
+> 2. Any _publicly available release_ of the AI SDKs must only contain _public API_ for AI Core features available in _production EU10_ under the service plan _extended_.
+> 3. E2E tests run against canary EU12 and production EU10 using test toggles. 
+
+Further explanations and notes:
+
+- Any features released exclusively under the `sap-internal` plan are not supported.
+  Similarly, any features released only to specific landscapes (other than prod EU10) are not supported.
+- There will be no releases of the SDKs to internal artifactory.
+- Please note that the following is allowed:
+  - Public API in an unreleased SDK version for unreleased AI Core features.
+    Notably, this will **block** the release of the SDK until the AI Core feature is released publicly.
+  - Internal code in a released SDK version for unreleased AI Core features.
+
+### An Example Development Lifecycle Iteration
+
+The following depicts a development flow where the AI SDK development steps are performed as soon as possible. 
+
+1. A new AI Core feature is being developed.
+2. A PR is raised on the AI SDK with a corresponding implementation, but so far not E2E-tested.
+   - Potentially aided by unit tests based on test data manually copied from e.g. Bruno.
+   - Generated code is created from a development version of the relevant spec file.
```

Comment:
The ADR first and foremost covers orchestration. So yes, it's relevant. And yes, we need the spec to be similar to prod. However, differences are not a blocker, because we are not merging the PR yet at this step. So the generated code doesn't need to be exact at this point.